src/main/webapp/WEB-INF/tags/student/feedbackSubmissionEdit/header.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="studentFeedbackSubmissionEdit.jsp - Display preview or moderation header" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="preview" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="moderation" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="name" required="true" %>
+<%@ attribute name="email" required="true" %>
+
+<c:choose>
+    <c:when test="${preview}">
+        <nav class="navbar navbar-default navbar-fixed-top">
+            <h3 class="text-center">Previewing Session as Student ${name} (${email})</h3>
+        </nav>
+    </c:when>
+    <c:otherwise>
+        <c:if test="${moderation}">
+            <nav class="navbar navbar-default navbar-fixed-top">
+                <h3 class="text-center">Moderating Responses for Student ${name} (${email})</h3>
```

Comment:
Considering nearly everything is the same, you can instead use the `c:choose` to set an attribute to "Previewing Session as" or "Moderating Responses for" instead.